### DISCUSSION TITLE
Clarifications about billing for plan "Organization"

### DISCUSSION THREAD

**Discussion by RTcarlos** [opengisch/QField#3836] [Q&A]
I'm Carlos Roberto, I live in Brazil, I work in a company that does reforestation.

My question is the following, under the charge of the new plans.

The Organization plan costs 16€/User/month (around 192€/year) correct?

Is that user responsible for creating the group or does each member have to be registered and pay an extra €16 per month?

My network would have between 15 and 20 people (1 editor and the rest would be collaborators).

We have a person responsible for creating the projects, and the others carry out the data collection, should these people be registered as well?

What would be the payment methods?

issue invoice?

No more for the moment, I await an answer.

**Comment by suricactus** [comment]
Hello, Carlos Roberto aka @RTcarlos . 

Your personal user (the one you login with) can own multiple organizations, and each organization has their own billing account, which  means separate invoices.

Any personal user, community or pro, can be a member of an organization. When a user is a member of an organization, the organization projects can be shared with them.

To add a new organization member, you need to have their username or email first. Which means they need to register their own users.

When QFieldCloud charges the monthly bill for your organizational account, you pay only for the active organization members. By active, it's someone who either downloaded or uploaded to a organizational project. If the organization member did nothing in during the month, it is not considered active.

Note there is a minimal amount of 2 users that will be billed each month, no matter if there were active.

Invoices are charged month by month, so your calculation is right: 16 EUR * 12 month = 192 EUR / year .

You can pay by credit/debit card, if you cannot do so, please contact us by email and we can find a solution.

| number of organization members | number of active users |  amount to pay |
|---------------------------------------------|---------------------------------|---------------------|
| 15 | 0 | 32 |
| 15 | 1 | 32 |
| 15 | 2 | 32 |
| 15 | 3 | 48 |
| 15 | 4 | 64 |
| ... | ... | ... |
| 15 | 15 | 240 |

Check my other answer here for more info:
https://github.com/opengisch/QField/discussions/3832#discussioncomment-4766947

Hope I managed to answer all your questions, feel free to ask more. :)
